packages/shared_preferences/shared_preferences_android/CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
-## 2.4.3
```

Comment:
I forget our rules here but I thought we couldnt roll back this version if it was published which the existence of an OOB change implies happened.

Reply:
@stuartmorgan please advise. Should we publish a new version?